routes/listproducts.js

```diff
@@ -4,10 +4,10 @@ const client = require('../database/db');
 var listProducts = express.Router();
 
 listProducts.get('/products', async (req, res) => {
-  const listProductsQuery = 'SELECT id, name, slogan, description, category, default_price FROM products LIMIT $1 OFFSET $2;';
+  const listProductsQuery = 'SELECT id, name, slogan, description, category, default_price FROM products WHERE id <= $1 AND id > $2';
```

Comment:
I'm surprised this makes it faster! However, it is sort of broken, because if a product is deleted, it will mess up. For example, if product 4 is deleted, you'll get product 1, 2, 3, and 5 (4 total), when what you really wanted was 5 total.